include/json/value.h

```diff
@@ -171,6 +171,9 @@ class JSON_API Value {
     CZString(ArrayIndex index);
     CZString(const char* cstr, DuplicationPolicy allocate);
     CZString(const CZString& other);
+#ifdef JSON_HAS_RVALUE_REFERENCES
+	CZString(CZString&& other);
+#endif
     ~CZString();
     CZString& operator=(CZString other);
```

Comment:
If you're providing a move ctor, you should also provide a move assignment.

    CZString& operator=(CZString&& other) noexcept;